test/hotspot/jtreg/containers/docker/JfrReporter.java

```diff
@@ -51,4 +51,3 @@ public static void main(String[] args) throws Exception {
         }
     }
```

Comment:
I'm not sure how a Java file ever got into the code base with trailing spaces, but a guess is that there have been a bug in jcheck that did not properly check for trailing whitespace at the end of the file, or something like that.